client/main.lua

```diff
@@ -82,9 +80,9 @@ CreateThread(function()
             end
 
             -- Hotwiring while in vehicle, also keeps engine off for vehicles you don't own keys to
-            if IsPedInAnyVehicle(ped, false) and not IsHotwiring then
+            if IsPedInAnyVehicle(cache.ped, false) and not IsHotwiring then
                 sleep = 1000
-                local vehicle = GetVehiclePedIsIn(ped)
+                local vehicle = GetVehiclePedIsIn(cache.ped)
```

Comment:
```suggestion
                local vehicle = cache.vehicle
```

Either use cache.vehicle or rename all `vehicle` to cache.vehicle to make it look a little cleaner